schema/objects/StockPlan.schema.json

```
@@ -34,13 +34,37 @@
       "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StockPlanCancellationBehaviorType.schema.json"
     },
     "stock_class_id": {
-      "description": "Identifier of the StockClass object this plan is composed of",
-      "type": "string"
+      "description": "[DEPRECATED in favor of stock_class_ids] Identifier of the StockClass object this plan is composed of.",
+      "type": "string",
+      "deprecated": "true"
+    },
+    "stock_class_ids": {
+      "description": "Identifiers of StockClass objects this plan is composed of",
+      "type": "array",
+      "items": {
```

Do we need to require the array to contain at least one ID, or is it allowed to be empty?

```suggestion
      "type": "array",
      "minItems": 1,
      "items": {
```

---

Yes, the array should contain at least one ID when stock_class_ids is provided